src/test/java/hw6/Test/JdiSteps.java

```diff
@@ -0,0 +1,66 @@
+package hw6.Test;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import hw6.DataFromJsonForMetalsColorPage;
+import hw6.JdiSite;
+import hw6.complexelements.HeaderMenuItems;
+import hw6.entities.User;
+import org.testng.Assert;
+
+import static hw6.JdiSite.getTitle;
+import static hw6.JdiSite.metalsColorsPage;
+import static hw6.entities.User.ROMAN;
+import static java.lang.String.format;
+
+public class JdiSteps {
+
+    public JdiSteps() {
+    }
+
+    public void openSite() {
+        JdiSite.open();
+    }
+
+    public void loginWasSuccessful(User user) {
+        JdiSite.login(ROMAN);
+        String actualUserName = JdiSite.homePage.getUserName();
+        Assert.assertEquals(actualUserName, ROMAN.getFullName(),
+                format(String.valueOf(ROMAN), actualUserName));
+        Assert.assertEquals(user.getFullName(), "ROMAN IOVLEV");
+    }
+
+    public void clickHeaderMenu(HeaderMenuItems metalsAndColors) {
+        JdiSite.clickItemOfHeaderMenu(metalsAndColors);
+    }
+
+    public void pageIsOpened(HeaderMenuItems metalsAndColors) {
+        metalsColorsPage.checkOpened();
+        Assert.assertEquals(getTitle(), "Metal and Colors", String.valueOf(metalsAndColors));
```

Comment:
Page::checkOpened method can be used in order to verify URL and Title as well, so no needs to create additional assert here. (Title should be specified in annotation for the page)